src/api/auth/token.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+from fastapi import HTTPException, Request, status
+
+from src import settings
+
+
+async def check_token(request: Request):
+    if not hasattr(settings, "ACCESS_TOKEN_FOR_PROCHARITY"):
+        return True
+    if not request.headers.__contains__("token"):
```

Comment:
Не нужно явно использовать дандер метод. Это строка будет под капотом вызывать `__contains__`:

```suggestion
    if "token" not in request.headers:
```

Reply:
Понял.